model/riscv_sys_control.sail

```diff
@@ -145,6 +147,18 @@ function getPendingSet(priv : Privilege) -> option((xlenbits, Privilege)) = {
   else None()
 }
 
+/* Check if a locally enabled interrupt is pending.
+ *
+ * This does not examine the global enable bits (MIE and SIE in
+ * mstatus) and their delegation in mideleg. It does honor the
+ * local interrupt enables in mie.
+ *
+ * This is used when the hart is in the Waiting state.
+ */
+function wakeForInterrupt() -> bool = {
```

Comment:
I'd probably call this `shouldWakeForInterrupt()`.

Reply:
Thats a better name, sure.